portal/src/pages/components/table.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import ReactMarkdown from "react-markdown";
+import { ComponentLayout, Changelog } from "../../components";
+import { renderer } from "../../presentation/markdownRenderer";
+import { TableExample } from "../../examples";
+
+// @ts-ignore
+import readmeContents from "!raw-loader!@fremtind/jkl-table/README.md";
+// @ts-ignore
+import changelog from "!raw-loader!@fremtind/jkl-table/CHANGELOG.md";
+// @ts-ignore
+import changelogReact from "!raw-loader!@fremtind/jkl-table-react/CHANGELOG.md";
+
+export default function TablePage() {
+    return (
+        <ComponentLayout title="Tabell" mainExample={<TableExample />} packageName="table">
+            <ReactMarkdown renderers={renderer} source={readmeContents} />
```

Comment:
Per above you could add a second example here with the `Example` component directly

Reply:
See comment above.

Reply:
You can insert a new `Example` component below the markdown if you want (you just have to export/import it). Something like:
```jsx
import { ComponentLayout, Changelog, Example } from "../../components";
import { Table } from "@fremtind/jkl-table-react";
import { secondExample } from "../../examples/TableExample";
// then, inside ComponentLayout
<h2 className="jkl-h3">Example with links</h2>
<Example exampleCode="secondExample" exampleComponents={{ Table }} />
```
A bit of a hassle, and sort of verbose, but it works :)

Reply:
Fixed!

Reply:
Fixed.